bundle/tests/scorecard/kuttl/probe/02-runtime-remove-probe.yaml

```diff
@@ -6,3 +6,4 @@ spec:
   # Add fields here
   livenessProbe: null
   readinessProbe: null
+  startupProbe: null
```

Comment:
For this scenario, we should check that startupProbe is no longer defined by adding it to 02-errors.yaml